index.html

```diff
@@ -0,0 +1,258 @@
+        >
+          My Resume
+        </button>
+        <div id="about-cards">
+          <div class="about-desc">
+            <img
+              class="about-icon"
+              src="./images/icons/about_icon1.svg"
+              alt="languages"
+            />
+            <h2 class="about-desc-font">Languages</h2>
+            <ul class="about-section-detail tags">
+              <li class="lists">RubyOnRails</li>
+              <li class="lists">HTML</li>
+              <li class="lists">CSS</li>
+              <li class="lists">JavaScript</li>
+            </ul>
+          </div>
+          <div class="about-desc">
+            <img
+              class="about-icon"
+              src="./images/icons/about_icon2.svg"
+              alt="framewords"
+            />
+            <h2 class="about-desc-font">Frameworks</h2>
+            <ul class="about-section-detail tags">
+              <li class="lists">RubyOnRails</li>
+              <li class="lists">Bootstrap</li>
+              <li class="lists">RSpec</li>
+              <li class="lists">Selenium</li>
+              <li class="lists">Selenium</li>
+            </ul>
+          </div>
+          <div class="about-desc">
+            <img
+              class="about-icon"
+              src="./images/icons/about_icon3.svg"
+              alt="Skills"
+            />
+            <h2 class="about-desc-font">Skills</h2>
+            <ul class="about-section-detail tags">
+              <li class="lists">GitHub</li>
+              <li class="lists">CodePen</li>
+              <li class="lists">GitLab</li>
+              <li class="lists">Terminal</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <footer id="contact-message">
+        <div id="footer-top">
+          <h2 id="footer-text" class="color-base">
+            I'm always interested in hearing about new projects, so if you'd like
+            to chat please get in touch.
+          </h2>
+          <form id="form" action="https://formspree.io/f/mrgjqpap" method="POST">
+            <label hidden>Name</label>
+            <input
+              class="form-fields"
+              type="text"
+              name="name"
+              id="name"
+              maxlength="30"
+              placeholder="Full Name"
+              required
+            />
+            <label hidden>Email Address</label>
+            <input
+              class="form-fields"
+              type="email"
+              name="Email Address"
+              id="email-address"
+              placeholder="Email Address"
+              required
+            />
+            <small id="email-invalidation-message"></small>
+            <label hidden>Subject</label>
+            <textarea
+              class="form-fields"
+              name="message"
+              id="message"
+              maxlength="500"
+              required
+            >
+      Describe you subject here...</textarea
+            >
```

Comment:
![image](https://user-images.githubusercontent.com/61361037/211076916-2d22afce-f53f-47b3-a2b6-3ffc42028fc7.png)

[OPTIONAL] The textarea has numerous spaces before and after the text is written. Remove that spaces, to make it look elegant.